frontend/src/components/ui/format-date.ts

```diff
@@ -31,9 +31,7 @@ export class FormatDate extends LitElement {
   @property() era?: "narrow" | "short" | "long";
 
   /** The format for displaying the year. */
-  // Technically "2-digit" is a valid option here too, but we've chosen not to
-  // use it, because it's sometimes ambiguous
-  @property() year?: "numeric";
+  @property() year?: "numeric" | "2-digit";
```

Comment:
Let's not use 2-digit years, we'd agreed on them being confusing. I think in the context of the logs we could probably leave off the date entirely, and keep it just visible in the tooltip, if we want the timestamp to take up less space?
```suggestion
  // Technically "2-digit" is a valid option here too, but we've chosen not to
  // use it, because it's sometimes ambiguous
  @property() year?: "numeric";
```

Reply:
That's fair, although I thought that all was resolved when we standardized localized formatting across the board?